integration-tests/test/native-eth.spec.ts

```diff
@@ -45,13 +45,13 @@ describe('Native ETH Integration Tests', async () => {
       const amount = utils.parseEther('0.5')
       const addr = '0x' + '1234'.repeat(10)
       const gas = await env.ovmEth.estimateGas.transfer(addr, amount)
-      expect(gas).to.be.deep.eq(BigNumber.from(21000))
+      expect(gas).to.be.deep.eq(BigNumber.from(0x35008e17cb6d))
```

Comment:
Decode and assert against the gas limit to decouple actual value being hardcoded

Reply:
I'm going to leave this out of scope for this PR and we should address the fact that the integration tests are coupled such that they all cannot run against kovan/goerli in an additional PR